### PR TITLE
changed background of test results '0'

### PR DESCRIPTION
**Pull request by mbtamuli**
Solving issue #799, this PR changes the background to blue color with a `label-primary`

### CODE REVIEW

File: bodhi/util.py [hunk 1]
```diff
@@ -364,7 +364,10 @@ def karma2html(context, karma):
         if karma < -2:
             cls = 'danger'
         else:
-            cls = 'success'
+            if karma == 0:
```
Comment (trishnaguha) [inline bodhi/util.py]: We can use `elif`  instead.

File: bodhi/util.py [hunk 2]
```diff
@@ -363,11 +363,10 @@ def karma2html(context, karma):
     if not cls:
         if karma < -2:
             cls = 'danger'
+        elif karma == 0:
+        	cls = 'primary'
```
Comment (trishnaguha) [inline bodhi/util.py]: Wrong indentation here.

File: bodhi/util.py [hunk 3]
```diff
@@ -363,8 +363,10 @@ def karma2html(context, karma):
...
         else:
-            cls = 'success'
+        	cls = 'success'
```
Comment (trishnaguha) [inline bodhi/util.py]: Wrong indentation here also.

**Comment by trishnaguha** [comment]
@mbtamuli LGTM! :+1: @lmacken will review it :).

**Comment by lmacken** [comment]
LGTM, thanks!

**Pull request merged**